docs-src/cloud/temporal-cloud-limits-sheet.md

```diff
@@ -0,0 +1,85 @@
+---
+slug: temporal-cloud-limits-sheet
+title: What are the Temporal Cloud default limits?
+sidebar_label: Temporal Cloud default limits
+description: This section describes many of the default settings and limits that apply to Workflow Executions in Temporal Cloud.
+tags:
+  - error
+  - warn
+  - limits
+---
+
+This section describes many of the default settings and limits that apply to Workflow Executions and do not pertain to the entire Temporal Cloud account or individual Namespaces in Temporal Cloud, unless otherwise specified.
+
+Some of these default limits are configurable by sending a [support ticket](/cloud/support#support-ticket) to our support team.
+
+### What is the maximum length for identifiers?
+
+Identifiers such as Workflow Id, Workflow Type, and Task Queue names have a maximum length of 1000 characters in UTF-8 format.
+
+This default limit is configurable by creating a [support ticket](/cloud/support#support-ticket).
+
+### What is the gRPC limit for each message received?
+
+Each gRPC message received has a limit of 4 MB.
+
+### What is the maximum limit for an Event batch size?
+
+The `DefaultTransactionSizeLimit` is configured at 4 MB, representing the maximum allowable transaction size for persisting Event Histories.
+
+### What are the default limits at the Namespace level?
+
+At the Namespace level, Temporal Cloud sets the following default limits:
+
+- Actions per second: 200 (with spikes to 400)
+- Certificates: 32 KB payload or 16 certificates, whichever is smaller
+- Concurrent Task pollers: 2,000 (configurable; maximum of 100,000)
+
+### What is the Concurrent limit?
+
+If the concurrent pending count of certain Commands exceeds 2,000, they will fail.
+These commands include:
+
+- `ScheduleActivityTask`
+- `SignalExternalWorkflowExecution`
+- `RequestCancelExternalWorkflowExecution`
+- `StartChildWorkflowExecution`
+
+This default limit is configurable by creating a [support ticket](/cloud/support#support-ticket).
+
+### What is the limit on the number of Signals received per Workflow Execution?
+
+The number of Signals received per Workflow Execution to 10,000.
+
+### What are the default limits for users and Namespaces at the account level?
+
+At the account level, Temporal Cloud sets the following default limits:
+
+- Users: 100
+- Namespaces: 10
+
+### What is the limit for Prometheus endpoint Retention Period?
+
+The Prometheus endpoint Retention Period is 30 days.
```

Comment:
is this the same retention period as workflow Retention Period? @tsurdilo do you know? and does it mean # of days of data i can see when hitting the endpoint? 

if changing the workflow Retention Period doesn't affect Prometheus, i'd change to:

```suggestion
### How much data does the Prometheus endpoint retain?

The Prometheus endpoint retains 30 days of data.
```